### PR TITLE
Docs: Separate parser and config questions in issue template

### PR DESCRIPTION
**Pull request by platinumazure**
Per discussion in chat, it seems like it might be worthwhile to separate the parser question from configuration. As it is, people will sometimes only include parser information.

I phrased the configuration line as a request rather than a question-- please let me know if I should change that.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ Note that leaving sections blank will make it difficult for us to troubleshoot a
 
 **What version of ESLint are you using?**
 
-**What configuration and parser (Espree, Babel-ESLint, etc.) are you using?**
+**What parser (Espree, Babel-ESLint, etc.) are you using?**
```
Comment (BYK) [inline .github/ISSUE_TEMPLATE.md]: I think we should use this opportunity to mention that Espree is the default one.

Comment (nzakas) [inline .github/ISSUE_TEMPLATE.md]: I'd maybe phrase it as, "What parser (default, Babel-ESLint, etc.) are you using?" The word "Espree" has little meaning to most ESLint users.

Comment (platinumazure) [inline .github/ISSUE_TEMPLATE.md]: @BYK @nzakas Good call, I'll modify accordingly. Thanks!

**Comment by platinumazure** [comment]
@BYK @nzakas Updated, very sorry for the delay.

**Comment by BYK** [comment]
LGTM. Leaving the merge and final stamp of approval to @nzakas.

**Pull request merged**